ruby3.2-fluentd15.yaml

```diff
@@ -1,14 +1,25 @@
 package:
-  name: ruby3.2-fluentd
+  name: ruby3.2-fluentd15
   version: 1.15.3
-  epoch: 1
+  epoch: 2
```

Comment:
hm. I think epoch 0 makes sense here since we are changing the package name

Reply:
Fair, it is technically a new package. Should I also withdraw the old fluentd packages since those will no longer be maintained?

Reply:
good question. probably shouldnt hurt